share/spice/forvo/forvo.js

```diff
@@ -12,143 +12,34 @@ function ddg_spice_forvo (api_result) {
     var script = $('[src*="/js/spice/forvo/"]')[0];
     var source = $(script).attr("src");
     var query = source.match(/forvo\/([^\/]+)\/\w+/)[1];
-
+    query = decodeURIComponent(query);
+    
     // Display the Spice plug-in.
     Spice.add({
-        data             : api_result,
-        name             : "Pronunciation",
-        sourceUrl       : "http://www.forvo.com/search/" + query,
-        sourceName      : "Forvo",
-        id              : "forvo",
-        template_frame   : "list",
-        templates : {
-            items         : api_result.items, //list,
-            item: Spice.forvo.forvo,
-            show          : 3,
-            max           : 5,
-            type          : "ul",
-            use_alternate_template: false
+        id: "forvo",
+        name: "Pronunciation",
+        data: {
+            list: api_result.items
         },
-        
-        
-    });
-
-    // This gets called when the sound is finished playing
-    // or when another player is playing. It basically just resets the look of the player.
-    var clearPlayer = function() {
-        var li = $(".spice2list_item");
-        li.removeClass("sm2_playing");
-        li.removeClass("sm2_paused");
-        li.addClass("sm2_stopped");
-        soundManager.stopAll();
-    };
-
-    // Loads and plays the audio file.
-    var isFailed = false;
-    var playSound = function(anchor) {
-        soundManager.stopAll();
-
-        if(isFailed) {
-            setTimeout(function() {
-                clearPlayer();
-            }, 1000);
-        } else {
-            var sound = soundManager.createSound({
-                id: anchor.attr("id"),
-                url: anchor.data("stream"),
-                onfinish: function() {
-                    clearPlayer();
-                }
-            });
-
-            sound.play();
-        }
-    };
-
-    // Initialize SoundManager2.
-    window.SM2_DEFER = true;
```

Comment:
I have problems to figure out where in the JS file I have to include the code refered to the audio button.

`this.playBtn = new DDG.Views.PlayButton({
    url: url,
    after: this.$el.find('.zci__def__pronunciation')
});`

Inside the Spice.add({}) section?